src/main/java/com/example/whathis/product/dto/request/ProductCreateRequest.java

```diff
@@ -0,0 +1,62 @@
+package com.example.whathis.product.dto.request;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Positive;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class ProductCreateRequest {
+
+    // 사업자 등록 번호
+    // 최초 상품 등록 시에만 필수, 이미 등록된 사용자는 생략 가능
+    // ex) 123-45-67890
+    @NotNull(message = "사업자등록번호는 필수입니다.")
+    @Pattern(
+        regexp = "^(\\d{3})-?(\\d{2})-?(\\d{5})$",
+        message = "사업자등록번호 형식이 올바르지 않습니다."
+    )
+    private String brn;
+
+    @NotBlank(message = "제품 사진은 필수입니다.")
+    @Pattern(
+        regexp = "^(https?://.*\\.(png|jpg|jpeg|webp))$",
+        message = "이미지 URL(jpg, jpeg, png, webp) 형식이어야 합니다."
+    )
+    private String thumbnailImageUrl;
+    
+    @NotBlank(message = "제목은 필수입니다.")
+    private String title;
+    
+    @NotBlank(message = "설명은 필수입니다.")
+    private String description;
```

Comment:
title과 description은 @Size를 통해 글자 수 제한을 두면 더 좋을 것 같습니다.
와디즈 정책을 참고하여 각 40자, 500자로 작성했습니다.
```suggestion
    @NotBlank(message = "제목은 필수입니다.")
    @Size(max = 40, message = "제목은 40자 이내로 입력해야 합니다.")
    private String title;
    
    @NotBlank(message = "설명은 필수입니다.")
    @Size(max = 500, message = "설명은 500자 이내로 입력해야 합니다.")
    private String description;
```